lib/s3_writer.rb

```diff
@@ -36,6 +36,11 @@ def feed_xml(checkouts)
         }
         xml.id "urn:nypl:item-checkout-feed"
         xml.updated Time.now
+        xml.tallies {
+          ItemTypeTally[:tallies].keys.each do |category|
```

Comment:
This generates XML like

```
<tallies>
  <Book_circulating>3</Book_circulating></tallies>
  <Other_circulating>3</Other_circulating></tallies>
  <Book_non-circulating>3</Book_non-circulating></tallies>
  ...
</tallies>
```

Firstly, can we use lowerCamelCase for elements to match the style of other elements in this atom feed? (e.g. "bookCirculating", "bookNonCirculating") Secondly, the absence of broader aggregations obligates the consumer to sum counts on their own. For example, a front-end interested in showing the total books in either collection has to look for the presence of and evaluate `//tallies/Book_circulating` and `//tallies/Book_non-circulating` as written. To look for the total circulating, one has to *know* what all of the coarse types are - or sum everything that matches `//tallies/*_circulating`. Perhaps that's fine. I wonder if it's worth helping the front-end out a little though?

Reply:
Also, I think we should give these made-up elements a `nypl` namespace at a minimum. They're not defined `Atom` elements, so we should prefix them with "nypl" (and declare the nypl namespace in the header same as we do for 'dc' and 'dcterms'

Reply:
I'll add the namespacing and the overall tallies. I'm a little unsure about lower camel case though. I want it to be clear where the item type ends and the collection type begins, keeping in mind there might be other types eventually reflected in this category.

Reply:
I've added overall counts. As per our discussion, I am now using "tally" as the tag name, and indicating more refined information using attributes. Also, these are correctly namespaced now.